src/styles/blocks/_header.scss

```
@@ -0,0 +1,104 @@
+.header {
+  background-size: cover;
+  background-position: center;
+  background-image: $bgrHeader;
+
+  &__title {
+    font-weight: $fontWeight700;
+    font-size: 2.25rem;
+    text-align: center;
+    line-height: 100%;
+    color: #292929;
+    letter-spacing: -1.5px;
+
+    @media (min-width: 426px) {
+      font-size: 3rem;
+      color: $DarkJungleGreen;
+    }
+
+    @include onTablet {
+      font-size: 4rem;
+      color: $DarkJungleGreen;
+    }
+
+    @include onDesktop {
+      font-size: 5rem;
+    }
+  }
+
+  &__content {
+    @extend %flexDirectionColumn;
+    justify-content: space-between;
+
+    box-sizing: border-box;
+    min-height: 480px;
+    padding: 29px 0 88px;
+
+    @include onTablet {
+      padding: 36px 0 130px;
+      min-height: 630px;
+    }
+
+    @include onDesktop {
+      min-height: 100vh;
+      padding: 36px 0 122px;
+    }
+  }
+
+  &__nav {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__icons {
+    display: flex;
+    transform: translateY(20%);
+  }
+
+  &__menu-opener {
+    background-image: url(../images/menu.svg);
+    transition: $transitionDuration ease;
+
+    &:hover {
+      background-image: url(../images/menu_hover.svg);
+    }
+  }
+
+  &__call {
+    margin-right: 30px;
+    text-decoration: none;
+    background-image: url(../images/phone.svg);
+
+    @include onTablet {
+      margin-right: 40px;
+    }
+
+    @include onDesktop {
+      &::before {
+        content: "+1 234 555-55-55";
+        position: relative;
+        top: 50%;
+        transform: translate(-100%, -50%);
+        display: block;
```

try avoid using relative on element which you try to position
```suggestion
        content: "+1 234 555-55-55";
        transform: translateX(-100%,);
        display: block;
        height: 24px;
```